### PR TITLE
Add FV test that silently cuts the etcd connection.

### PR DESCRIPTION
**Pull request by fasaxc**
## Description

- Use the raw table to silently drop packets from one connection.
- Check that felix detects and reconnects.

## Todos
- [x] Unit tests (full coverage)
- [x] Integration tests (delete as appropriate) In plan/Not needed/Done
- [x] Documentation
- [ ] Backport to 3.0.x
- [x] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: fv/etcd_restart_test.go [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+			var portRegexp = regexp.MustCompile(`sport=(\d+).*dport=2379`)
+			for _, felix := range felixes {
+				// Use conntrack to identify the source port that Felix is using.
+				out, err := felix.ExecOutput("conntrack", "-L")
+				Expect(err).NotTo(HaveOccurred())
+				logrus.WithField("output", out).WithError(err).Info("Conntrack entries")
+				found := false
+				for _, line := range strings.Split(out, "\n") {
+					matches := portRegexp.FindStringSubmatch(line)
+					if len(matches) == 0 {
```
Comment (neiljerram) [inline fv/etcd_restart_test.go]: Looks like "< 2" would be a more relevant check, given that code below uses `matches[1]`.

File: fv/etcd_restart_test.go [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+						continue
+					}
+					found = true
+
+					// Use the raw table to drop the single TCP connection that felix is using,
```
Comment (neiljerram) [inline fv/etcd_restart_test.go]: You say "single", but this loop will handle and blackhole multiple connections to 2379, I believe.  I'm unclear whether that is (a) OK, or (b) your intention.

File: fv/etcd_restart_test.go [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+						"-d", etcd.IP,
+						"-m", "multiport", "--source-ports", matches[1],
+						"-j", "DROP")
+				}
+				Expect(found).To(BeTrue(), "Failed to detect any felix->etcd connections")
+				felix.Exec("conntrack", "-D", "--orig-dst", etcd.IP)
+			}
+		})
+
+		By("starting to update policy again", func() {
```
Comment (neiljerram) [inline fv/etcd_restart_test.go]: "starting to"?  I think this should just be "updating policy again".

File: fv/etcd_restart_test.go (hunk 2 above)
Comment (fasaxc) [reply fv/etcd_restart_test.go]: Intentional to sever any and all connections.  (Quite possible that gRPC uses more than one, if it so chooses.)

**Pull request merged**